Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM arm64v8/python:latest as base
```

Comment:
I had issues with pulling this image, but it's ok if it works for you. I changed it to a generic python image and could get the container up.

Just something to keep in mind if it's becoming a problem in future exercises.

Reply:
Thanks Bogdan. I had to use this image because I'm on Apple Silicon so needed a specific architecture